src/main.cpp

```diff
@@ -150,7 +150,11 @@ int main(int argc, char *argv[])
     IPC& ipc = IPC::getInstance();
 #endif
 
-    sodium_init(); // For the auto-updater
+    if (sodium_init() == -1) // For the auto-updater
```

Comment:
comparing for `== -1` is technically correct, but I think it is better to compare for `< 0` if there are error codes added in the future.